rust/operator-binary/src/controller.rs

```diff
@@ -1213,9 +1213,14 @@ async fn build_node_rolegroup_statefulset(
         })
         .add_empty_dir_volume(
             "log",
-            Some(product_logging::framework::calculate_log_volume_size_limit(
-                &[MAX_NIFI_LOG_FILES_SIZE, MAX_PREPARE_LOG_FILE_SIZE],
-            )),
+            // Set volume size to higher than theoretically necessary to avoid running out of disk space as log rotation triggers are only checked by Logback every 5s.
+            Some(
+                MemoryQuantity {
+                    value: 500.0,
+                    unit: BinaryMultiple::Mebi,
+                }
+                .into(),
+            ),
```

Comment:
nit: We try to stay below 120 chars width :)

I personally would prefer to not "miss-use" the `MemoryQuantity` struct for disk space

```suggestion
            // Set volume size to higher than theoretically necessary to avoid running out of disk space as log rotation
            // triggers are only checked by Logback every 5s.
            Some(Quantity("500Mi".to_string())),
```

Reply:
> I personally would prefer to not "miss-use" the MemoryQuantity struct for disk space

`MemoryQuantity` is meant to be used for volatile as well as non-volatile memory:

https://github.com/stackabletech/operator-rs/blob/stackable-operator-0.74.0/crates/stackable-operator/src/memory.rs#L185

Reply:
Alright, please disregard my comment than, sorry :see_no_evil: